client/src/Utils/getToken.ts

```diff
@@ -1,4 +1,4 @@
-const getToken = () => {
```

Comment:
I see two times this file, one in "Utils" and another in "utils", delete one?

Reply:
that is a git issue after I capitalised the folder name to make it unified with other naming. I used git rm --cached <file_name> for all the files in the deprecated folder and committed changes. It seems it has worked and removed tracking. Let me know if you still see it somewhere.